lite/examples/pose_estimation/android/build.gradle

```diff
@@ -0,0 +1,26 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+buildscript {
+    ext.kotlin_version = "1.4.32"
+    repositories {
+        google()
+        jcenter()
+    }
+    dependencies {
+        classpath "com.android.tools.build:gradle:4.1.3"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+
+        // NOTE: Do not place your application dependencies here; they belong
+        // in the individual module build.gradle files
+    }
+}
+
+allprojects {
+    repositories {
+        google()
+        jcenter()
```

Comment:
Please refer to https://github.com/tensorflow/examples/blob/730a2ff0f72cbb8baf98daa7a676fe9b7a330ec4/lite/examples/image_classification/android/build.gradle#L20 for repositories.